R/RLearner_classif_RRF.R

```diff
@@ -49,3 +49,28 @@ predictLearner.classif.RRF <- function(.learner, .model, .newdata, ...) {
   p = predict(object = .model$learner.model, newdata = .newdata, type = type, ...)
   return(p)
 }
+
```

Comment:
Don't we need an `#'export` here to register the S3 method in the NAMESPACE?
(The same for all other learners.)